eclipse-theia/theia
# repo: upgrade `yarn.lock`

#### What it does


The commit updates the repository's `yarn.lock` to better represent what downstream extensions and applications may pull given  our dependency version ranges.

#### How to test


- It should build and CI should pass 
- The application should successfully start and operate (manual smoke tests)

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
